src/fluree/db/ledger/mutable.clj

```diff
@@ -42,10 +44,10 @@
 
 (defn purge-block
   [conn nw ledger block flakes]
-  (go-try (let [conn-rename   (:storage-rename conn)
+  (go-try (let [conn-delete   (:storage-delete conn)
                 block-key     (storage/ledger-block-key nw ledger block)
                 current-block (<? (storage/read-block conn nw ledger block))
-                _             (<?? (conn-rename block-key (str block-key "-delete")))
+                _             (<?? (conn-delete block-key))
```

Comment:
Is it okay to use a blocking take inside a go block? The docstring for `<!!` says "Not intended for use in direct or transitive calls from (go ...) blocks", but I'm not totally familiar with this case.

Reply:
I believe that the original contributor's intent was for the rename/delete operation to be blocking.  The purge operation should fail if even one block could not be purged.

We probably need to test a bit more to verify that the blocking actually happens.

Reply:
I tried a basic example at the repl and it seemed to work, but I don't know what the consequences are or why that warning is there.

Reply:
https://fluree.atlassian.net/browse/FC-1369